src/components/social.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import Social from './social.js';
+import reactGA from 'react-ga';
+
+var ShareProgressButton = React.createClass({
+  componentDidMount: function() {
+    var shareProgressHolder = document.querySelector(".share-progress-holder");
+    var shareProgressButton = shareProgressHolder.querySelector("." + this.props.shareProgress);
+    if (shareProgressButton) {
+      shareProgressHolder.removeChild(shareProgressButton);
+      this.shareProgressContainer.appendChild(shareProgressButton);
+    }
+  },
+  componentWillUnmount: function() {
+    var shareProgressButton = this.shareProgressContainer.querySelector("." + this.props.shareProgress);
+    var shareProgressHolder = document.querySelector(".share-progress-holder");
+    if (shareProgressButton) {
+      this.shareProgressContainer.removeChild(shareProgressButton);
+      shareProgressHolder.appendChild(shareProgressButton);
+    }
+  },
+  render: function() {
+    return (
+      <div onClick={this.props.onClick} className="share-progress-wrapper">
+        <div ref={(container) => { this.shareProgressContainer = container; }}></div>
+        {this.props.children}
+      </div>
+    );
+  }
+});
+
+var SocialButton = React.createClass({
+  render: function() {
+    return (
+      <a onClick={this.props.onClick} href={this.props.href} target="_blank">
```

Comment:
This feels like a straight passthrough, in which case we can use `<a {...this.props} target="_blank"/>`